### DISCUSSION TITLE
Feedback: Codespaces Build Screen May Cause Confusion

### DISCUSSION THREAD

**Discussion by startakovsky** [community/community#9491] [Codespaces]
Sample screen during Codespaces build. If a user is unfamiliar nor doesn't have a good understanding then reading lines 2 and 3 in that order seems quite hard to reconcile. See below.

1. Image found.
2. Container built.
3. Building container . . .

**Comment by tanmayeekamath** [comment]
Hey @startakovsky, just tried creating one on my end and this is what I saw: 
<img width="279" alt="image" src="https://user-images.githubusercontent.com/30297258/148111060-5ad52ecd-e2e2-4b2f-b2ca-eff44697eac1.png">

Are you seeing this for every codespace create? Trying to understand if this could've been an error that we need to look into. Thanks for the feedback.

-- Tanmayee

**Reply by startakovsky** [reply]
@tanmayeekamath The way I would reproduce it is to invoke building a container from scratch. When that happens you'll see the third line swapped with "Connecting..."

**Comment by jkeech** [comment]
Each message has an "in progress" and "finished" state, and the wording changes slightly when it's done. For instance, the first step goes from "Finding image..." to "Image found.". The second step goes from "Building container..." to "Container built.". And then the third step goes to "Connecting..." when the container is built and it's available for connection.

https://user-images.githubusercontent.com/746020/150625924-9ee08fe4-743a-4fc1-a5b0-b1c835f570e8.mov

I think there's a bug in the rebuild flow though where it doesn't clear all of the previous lines when it goes back to that terminal screen while the codespace is rebuilding. @startakovsky, when you saw this, was it during a rebuild or during the initial codespace creation?
